packages/shared-ux/chrome/serverless_projects_documentation.md

```diff
@@ -0,0 +1,289 @@
+| `recentlyAccessed$`  | `Observable<RecentItem[]>` | An optional observable for recently accessed items. If not provided, the recently accessed items from the Chrome service will be used. |
+| `defaultIsCollapsed` | `boolean`                  | If set to `true`, the recently accessed list will be collapsed by default. The default value is `false`.                               |
+
+### React components
+
+If you need other navigation sections in your navigation you will need to use our React components. They have the same properties as seen above with the exception of the `unstyled` prop that we will detail below.
+
+```tsx
+import { NavigationKibanaProvider, Navigation } from '@kbn/shared-ux-chrome-navigation';
+
+const createServerlessSearchSideNavComponent =
+  (
+    core: CoreStart,
+    { serverless, cloud }: { serverless: ServerlessPluginStart; cloud: CloudStart }
+  ) =>
+  () => {
+    return (
+      <NavigationKibanaProvider core={core} serverless={serverless} cloud={cloud}>
+        <Navigation>
+          <Navigation.Group id="group1" title="Group 1" defaultIsCollapsed={false}>
+            <Navigation.Item id="item1" title="Item 1" />
+            <Navigation.Item id="item2" title="Item 2" />
+            <Navigation.Item id="item3" title="Item 3" />
+          </Navigation.Group>
+          
+          <div>
+            <p>Any other section you might need</p>
+          </div>
+
+          <Navigation.Group id="group2" title="Group 2">
+            <Navigation.Item id="item1">
+              <EuiText color="blue">Title can also be a React node</EuiText>
+            </Navigation.Item>
+
+            <Navigation.Group id="nestedGroup" title="Group can be nested">
+              <Navigation.Item id="item1" title="Item 1" />
+            </Navigation.Group>
+          </Navigation.Group>
+        </Navigation>
+      </NavigationKibanaProvider>
+    );
+  };
+```
+
+And as with the tree definition above you will have to set the navigation component on the `serverless` plugin.
+
+```ts
+// plugin.ts (public) "start()"
+serverless.setSideNavComponent(createComponent(core, { serverless, cloud }));
+```
+
+#### `unstyled`
+
+If you want to completely customize your UI and just need to declare you navigation tree you can pass the `unstyled` property to your `<Navigation />`.
+
+```tsx
+/**
+ * This JSX will correctly declare your tree structure but will not have any UI applied.
+ Tree generated:
+[{
+  id: 'my-group',
+  title: "My Group,
+  children: [{
+    id: 'item-1',
+    title: 'Item 1',
+  }, {
+    id: 'item-2',
+    title: 'Item 2',
+  }]
+}]
+ */
+<Navigation unstyled>
+  <Navigation.Group id="my-group" title="My group">
+    <Navigation.Item id="item-1" title="Item 1">
+      <div>Your custom UI</div>
+    </Navigation.Item>
+    <Navigation.Item id="item-2" title="Item 2">
+      <div>Your custom UI</div>
+    </Navigation.Item>
+  </Navigation.Group>
+</Navigation>
+```
+
+## Breadcrumbs
+
+TODO
+
+## Toolbar
```

Comment:
@tsullivan, is there something specific we need to cover for Toolbar and Search Bar? I don't think so, but might be missing something